### DISCUSSION TITLE
Add car battery %

### DISCUSSION THREAD

**Discussion by Ironrampant** [slipx06/sunsynk-power-flow-card#523] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![image](https://github.com/user-attachments/assets/efdf0e6b-2d1d-4699-a773-afe8d3ac3f4d)

In my example picture the power to the car is replicated in 2 locations (7.1 kW in this case). The "non-essential" text below is also wasted space, as the car icon shows what it is.

### Expected behaviour

I'd like to replace something with the car battery %

### Possible Solutions

 Maybe an option to use the lower box if there is only one non-essential?  Or maybe swapping out the non-essential text field?

### Mode

lite

### Context / Reason

Utilises space better and reduces the need for having the car battery percent on other parts of the screen

**Comment by slipx06** [comment]
Hi. You could set `additional_loads: 0` to remove the duplicate box assuming there is nothing else connected to the non-essential circuit and set the non essential icon; `nonessential_icon: mdi:car`. You can also remove the Non Essential name or set is to something else; `nonessential_name: " "` 

![image](https://github.com/user-attachments/assets/27998e69-c464-4b3c-9624-567b3d3aa4ac)

Alternatively you can use the `non_essential_load1_extra: <battery_sesnor>` to show additional information. You can still remove the non essential name to free up space but you will have the duplicate data if nothing else is connect to your non essential circuit

![image](https://github.com/user-attachments/assets/b4c10f33-a6e8-4650-88d3-56b638ccfb6d)

The final option is to use the AUX input to show this

![image](https://github.com/user-attachments/assets/1cec6ff3-beb6-47b1-8529-5a12940eb45f)

Have a look at these sensors
```
aux_name: Car
aux_daily_name: DAILY CAR
aux_type: mdi:car
aux_power_166: input_number.test_number
aux_load1_extra: sensor.sunsynk_battery_capacity_shutdown
```

**Comment by slipx06** [comment]
Also looks like you are using an old version of the card. You should update it as some things have changed

**Comment by alphabravocharly72** [comment]
Hi,
I prefer the compact or light view. I tried to add the Bat SOC with the non_essential_load1_extra. But it doesnt work.
In full mode it works.
How can I bring the % over the icon?
Thx
![image](https://github.com/user-attachments/assets/5c0220d6-8085-4cde-8c71-8924ec5f4f6a)
